packages/gatsby-plugin-react-helmet/README.md

```diff
@@ -41,5 +41,5 @@ If you are using this plugin with React hooks, you may notice some errors like `
 
 ## Examples
 
-- [GatsbyJS.org](https://github.com/gatsbyjs/gatsby/blob/master/www/src/components/site-metadata.js)
+- [GatsbyJS.org](https://github.com/gatsbyjs/gatsby/blob/pieh-www-tests-master/www/src/components/site-metadata.js)
```

Comment:
I think we should just remove `gatsby.org` example for now. We can replace it with something else. Let's use `kentcdodds.com`:

```suggestion
- [kentcdodds.com](https://github.com/kentcdodds/kentcdodds.com/blob/master/src/components/seo/index.js)
```